build.rs

```diff
@@ -0,0 +1,28 @@
+use std::process::Command;
+
+fn run_command_with_fallback(cmd: &str, args: &[&str], fallback: &str) -> String {
+    Command::new(cmd)
+        .args(args)
+        .output()
+        .ok()
+        .and_then(|o| String::from_utf8(o.stdout).ok())
+        .map(|s| s.trim().to_string())
+        .filter(|s| !s.is_empty())
+        .unwrap_or_else(|| fallback.to_owned())
+}
+
+fn main() {
+    let git_tag = run_command_with_fallback("git", &["describe", "--tags", "--abbrev=0"], "none");
+
+    let git_commit = run_command_with_fallback("git", &["rev-parse", "HEAD"], "none");
+
+    let dockerfile_hash = run_command_with_fallback(
```

Comment:
`dockerfile_hash` on it's own does not provide a llot of info. Need to decided if needed and how it is used.